eth2near/logger/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "logger"
```

Comment:
```suggestion
name = "eth2near-logger"
```

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/825/commits/4683dcd634d532d10f8c6f5b29479f1e224be26e